rust-checks/src/bin/missing-patch-comment.rs

```diff
@@ -84,7 +84,7 @@ fn process_patch_list(
 
             report.push(NixpkgsHammerMessage {
                 msg:
-                    "Please add a comment on the line above, explaining the purpose of this patch."
+                    "Consider add a comment on the line above explaining the purpose of this patch."
```

Comment:
“add” → “adding”

And maybe keep the comma, or move “on the line above” to the end and change “above” to “preceding”.